omero/sysadmins/unix/install-web.txt

```diff
@@ -176,3 +176,7 @@ The most popular configuration options include:
 -  Customizing index page, see :property:`omero.web.index_template`.
 
 -  Enabling a public user see :doc:`/sysadmins/public`.
+
+.. Note:: Depending on how your OMERO.web server is deployed, you may need to
```

Comment:
I would be more explicit. something like: depending on the deployment choice you either have to restart OMERO.web using `bin/omero web restart` or reload Apache